### PR TITLE
added a blog

### PR DESCRIPTION
**Pull request by chenj-freedom**
Fixes # .

Make sure these boxes are checked before creating your pull request -- thank you!

- [x] Add Blog Header, including title, author, date.  
    For example:  

    ``` md
    ---
    title: "Your Great Blog Title"
    author: your_github_id
    image: /assets/2020/08-your-blog-slug/teaser.jpg
    ---

    YOUR_FIRST_SENTENCE_SHOULD_BE_A_SUMMARY_FOR_YOUR_POST
    ```

- [x] Keep all filenames & url as lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor` instead of `2017-10-06-WeChat PC Impactor`,
- [x] Embed the photo & video before publishing, save all external file to a folder named by `${MONTH}-${YOUR_BLOG_SLUG}/` under `/assets/2020` directory.
- [x] Add teaser image for the blog
- [x] Add your self as an author to the author data file at <https://github.com/wechaty/wechaty.js.org/blob/master/_developers/YOUR_GITHUB_USERNAME.md> with your GitHub username. And please also remember to reference yourself with the Github username in your blog post, so that we can have nice author information besides your blog after we published it!
  For excample:
  ```md
  ---
  name: Jiarui LI (李佳芮)
  site: https://lijiarui.github.io
  bio: 句子互动创始人 & CEO，微软人工智能最具价值专家 (AI MVP)
  avatar: /assets/developers/lijiarui/avatar.png
  email: rui@juzi.bot
  twitter: https://weibo.com/u/2175505900
  ---
  ```
  Here is the example file:
  - lijiarui.md: https://github.com/wechaty/wechaty.github.io/blob/master/_developers/lijiarui.md
  - It will shown like this on wechaty blog: https://wechaty.github.io/developers/lijiarui/

To read more about the contribute guideline, visit: <https://github.com/wechaty/wechaty.js.org#contribute-guideline>

### CODE REVIEW

**Comment by chenj-freedom** [comment]
1

File: jekyll/_developers/chenj-freedom.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+name: chenj-freedom
```
Comment (lijiarui) [inline jekyll/_developers/chenj-freedom.yml]: You should change this file from `chej-freedom.yml` to `chej-freedom.md`

File: jekyll/_posts/2020-09-14-my-first-wechaty-bot.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+title: /assets/2020/my-first-wechaty-bot/2020-09-my-first-wechaty-bot.png
+---
+
+手头上的事情越来越多，回复微信都是件很麻烦的事，对于IT行业的人来说，能用机器解决的事就不应该手动去做，于是就有了构建微信机器人的想法。
+
+## 技术选型
+
+首先，自然是对底层架构的选型，在网上也查找了很久，也尝试了许多其他的开源库，很多前两年（今天是2020年9月12日）的微信机器人库都已经过期无法使用了，主要原因他们都是基于web微信的协议，而腾讯的安全策略屏蔽了web微信协议。所以，经过一番挑挑拣拣，发现网上也就剩下最后一个可用的库，那就是这篇文章的主角，wechaty，官方github地址：[https://github.com/wechaty](https://github.com/wechaty)。
+
+这是一个商业公司（北京句子科技有限公司）开发的一套半开源半商业的库，基于web微信协议的库确实都开源了，但是基于上述腾讯的政策，web微信现在基本是都用不了了，所以web微信这套开源库，也就只能学学而已。但是他们还开发了一套基于pad微信协议的库，github地址：[https://github.com/wechaty/wechaty-puppet-padplus](https://github.com/wechaty/wechaty-puppet-padplus)。这个可以用，但是呢，需要token，每个月收费200RMB，不过可以试用，15天的试用期，15天后，有机会把token试用期最长延期到1年，需要写博客支持他们发起的一个开源计划，说白了，就是替他们公司宣传这个库，只能说各取所需吧。这个开源计划可以参考：[https://github.com/juzibot/Welcome/wiki/Everything-about-Wechaty](https://github.com/juzibot/Welcome/wiki/Everything-about-Wechaty)。
```
Comment (lijiarui) [inline jekyll/_posts/2020-09-14-my-first-wechaty-bot.md]: Please change `北京句子科技有限公司` to `北京句子互动科技有限公司`

File: jekyll/_posts/2020-09-14-my-first-wechaty-bot.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+
+## 了解原理
+
+开发之前呢，最好还是了解下这个架构的原理，具体的不细说，有兴趣的github上都可以找到答案。这里就放一张官方的原理图。
+
+![image](https://upload-images.jianshu.io/upload_images/24686309-5961452efa67c6f9.png?imageMogr2/auto-orient/strip%7CimageView2/2/w/1240)
```
Comment (lijiarui) [inline jekyll/_posts/2020-09-14-my-first-wechaty-bot.md]: Please localize this file in order to a future 404 error.

File: jekyll/_posts/2020-09-14-my-first-wechaty-bot.md [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+![image.png](https://upload-images.jianshu.io/upload_images/24686309-0df665d0b9112bab.png?imageMogr2/auto-orient/strip%7CimageView2/2/w/1240)
+之后就可以手机微信扫码登录，当别人给你当前登录的微信号发送消息“12345”，就会自动回复“hello world”。
+
+## 遇到的问题
+
+开发之中也是碰到了一些问题，其实都可以从github官方的issues里面找到答案，我这边记录下我遇到的问题。
+
+### cmd窗口二维码显示扫码不完善
+
+![image.png](https://upload-images.jianshu.io/upload_images/24686309-08c59775ef5d4ae1.png?imageMogr2/auto-orient/strip%7CimageView2/2/w/1240)
```
Comment (lijiarui) [inline jekyll/_posts/2020-09-14-my-first-wechaty-bot.md]: Please localize this file in order to a future 404 error.

**Review by lijiarui — CHANGES_REQUESTED** [review]
Please change as I request.

File: jekyll/_developers/chenj-freedom.yml (hunk 1 above)
Comment (chenj-freedom) [reply jekyll/_developers/chenj-freedom.yml]: fixed

File: jekyll/_posts/2020-09-14-my-first-wechaty-bot.md (hunk 2 above)
Comment (chenj-freedom) [reply jekyll/_posts/2020-09-14-my-first-wechaty-bot.md]: fixed

File: jekyll/_posts/2020-09-14-my-first-wechaty-bot.md (hunk 3 above)
Comment (chenj-freedom) [reply jekyll/_posts/2020-09-14-my-first-wechaty-bot.md]: fixed

File: jekyll/_posts/2020-09-14-my-first-wechaty-bot.md (hunk 4 above)
Comment (chenj-freedom) [reply jekyll/_posts/2020-09-14-my-first-wechaty-bot.md]: fixed

File: jekyll/_developers/chenj-freedom.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+name: chenj-freedom
+site: https://github.com/chenj-freedom
+avatar: /assets/developers/chenj-freedom/avatar.png
```
Comment (lijiarui) [inline jekyll/_developers/chenj-freedom.md:4]: Where is this file?

**Review by lijiarui — CHANGES_REQUESTED** [review]
Please change as I request.

File: jekyll/_developers/chenj-freedom.md (hunk 5 above)
Comment (chenj-freedom) [reply jekyll/_developers/chenj-freedom.md:4]: fixed

File: jekyll/_posts/2020-09-14-my-first-wechaty-bot.md [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+
+首先，自然是对底层架构的选型，在网上也查找了很久，也尝试了许多其他的开源库，很多前两年（今天是2020年9月12日）的微信机器人库都已经过期无法使用了，主要原因他们都是基于web微信的协议，而腾讯的安全策略屏蔽了web微信协议。所以，经过一番挑挑拣拣，发现网上也就剩下最后一个可用的库，那就是这篇文章的主角，wechaty，官方github地址：[https://github.com/wechaty](https://github.com/wechaty)。
+
+这是一个商业公司（北京句子互动科技有限公司）开发的一套半开源半商业的库，基于web微信协议的库确实都开源了，但是基于上述腾讯的政策，web微信现在基本是都用不了了，所以web微信这套开源库，也就只能学学而已。但是他们还开发了一套基于pad微信协议的库，github地址：[https://github.com/wechaty/wechaty-puppet-padplus](https://github.com/wechaty/wechaty-puppet-padplus)。这个可以用，但是呢，需要token，每个月收费200RMB，不过可以试用，15天的试用期，15天后，有机会把token试用期最长延期到1年，需要写博客支持他们发起的一个开源计划，说白了，就是替他们公司宣传这个库，只能说各取所需吧。这个开源计划可以参考：[https://github.com/juzibot/Welcome/wiki/Everything-about-Wechaty](https://github.com/juzibot/Welcome/wiki/Everything-about-Wechaty)。
+
+## 了解原理
+
+开发之前呢，最好还是了解下这个架构的原理，具体的不细说，有兴趣的github上都可以找到答案。这里就放一张官方的原理图。
+
+![image](jekyll/assets/2020/my-first-wechaty-bot/1.png)
```
Comment (lijiarui) [inline jekyll/_posts/2020-09-14-my-first-wechaty-bot.md]: Please change to `/assets/2020/my-first-wechaty-bot/1.png`

File: jekyll/_posts/2020-09-14-my-first-wechaty-bot.md [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+![image.png](jekyll/assets/2020/my-first-wechaty-bot/2.png)
+之后就可以手机微信扫码登录，当别人给你当前登录的微信号发送消息“12345”，就会自动回复“hello world”。
+
+## 遇到的问题
+
+开发之中也是碰到了一些问题，其实都可以从github官方的issues里面找到答案，我这边记录下我遇到的问题。
+
+### cmd窗口二维码显示扫码不完善
+
+![image.png](jekyll/assets/2020/my-first-wechaty-bot/3.png)
```
Comment (lijiarui) [inline jekyll/_posts/2020-09-14-my-first-wechaty-bot.md]: Please change to `/assets/2020/my-first-wechaty-bot/1.png`

File: jekyll/_posts/2020-09-14-my-first-wechaty-bot.md [hunk 8]
```diff
@@ -0,0 +1,138 @@
...
+```
+
+然后执行命令如下命令运行：
+
+```bash
+node bot.js
+```
+
+成功运行：
+![image.png](jekyll/assets/2020/my-first-wechaty-bot/2.png)
```
Comment (lijiarui) [inline jekyll/_posts/2020-09-14-my-first-wechaty-bot.md]: Please change to `/assets/2020/my-first-wechaty-bot/2.png`

**Review by lijiarui — CHANGES_REQUESTED** [review]
Please change as I request.

File: jekyll/_posts/2020-09-14-my-first-wechaty-bot.md (hunk 6 above)
Comment (chenj-freedom) [reply jekyll/_posts/2020-09-14-my-first-wechaty-bot.md]: fixed

File: jekyll/_posts/2020-09-14-my-first-wechaty-bot.md (hunk 8 above)
Comment (chenj-freedom) [reply jekyll/_posts/2020-09-14-my-first-wechaty-bot.md]: fixed

File: jekyll/_posts/2020-09-14-my-first-wechaty-bot.md (hunk 7 above)
Comment (chenj-freedom) [reply jekyll/_posts/2020-09-14-my-first-wechaty-bot.md]: fixed

**Comment by chenj-freedom** [comment]
不继续审核了吗？  一个月多的PR了

**Comment by huan** [comment]
Refer to #582

**Review by huan — APPROVED** [review]
LGTM

**Comment by huan** [comment]
Thank you very much for sharing this great post, the blog is online with URL <https://wechaty.js.org/2020/09/14/my-first-wechaty-bot>

> You've invited chenj_freedom to Wechaty! They'll be receiving an email shortly. They can also visit https://github.com/wechaty to accept the invitation.

1. I have invited you to join the Wechaty Contributor Team on our GitHub Wechaty Organization, please accept it by visiting <https://github.com/wechaty>
1. We have a WeChat room for Wechaty Contributors only, if you'd like to join, please contact @lijiarui on WeChat.

**Pull request merged**